lib/segment/src/spaces/mod.rs

```diff
@@ -8,6 +8,7 @@ pub mod simple_sse;
 #[cfg(target_arch = "x86_64")]
 pub mod simple_avx;
 
+pub mod metric_f16;
```

Comment:
Reorganise please the `mod` like `metric_unit`. `half`-typed vectors should have the same design as `byte`-typed vectors.
Please move all implementations like in this folder:
https://github.com/qdrant/qdrant/tree/dev/lib/segment/src/spaces/metric_uint

Reply:
Done